qa/1589.out

```diff
@@ -2,11 +2,15 @@ QA output created by 1589
 pcp2json invocation
 pcp2json HTTP POST (sorted):
 
+
+
+
+
 		"value":NCPU
 	      "ncpu": {
 	      }
 	    "@interval": "0",
-	    "@timestamp":SECS,
+	    "@timestamp": "2024-02-14 18:31:30",
```

Comment:
Something strange happened here and we no longer seem to be filtering this timestamp field?